apis/Google.Cloud.Firestore/Google.Cloud.Firestore.Tests/FilterTest.cs

```diff
@@ -48,6 +48,10 @@ public void Where_EqualTo_String()
         Assert.Equal(expected, query.ToStructuredQuery());
     }
 
+    [Fact]
+    public void Where_EqualTo_NonSerilizableValue() =>
```

Comment:
I'd rename this to `Where_UnserializableFilter`. The "equal to" part isn't particularly interesting... it's that the filter, whatever it represents, can't be serialized.